game-app/game-core/src/test/java/org/triplea/live/servers/FetchingCacheTest.java

```diff
@@ -50,13 +51,15 @@ void setUp() {
             .build();
   }
 
+  @Disabled
```

Comment:
There is more to do.. we now get the latest version from 'ClientSettings' with a default of 'prod.triplea-game.org'

We'll want to send a header with a triplea-version. Later we can configure nginx to forward old versions to an old lobby.